source/tutorials/mirror-upstream-server.rst

```diff
@@ -0,0 +1,237 @@
+.. _mirror-upstream-server:
+  
+Set Up a Mirror of the |CL| Upstream Update Server
+##################################################
+
+This tutorial shows how to set up a mirror of the upstream |CL| server
+and configure your |CL| clients to use it.
```

Comment:
Could you briefly identify the target audience/use case in another sentence?  It could be something similar to: 
```
System administrators or IT professionals often need to configure update servers...
```